integrations/chainlit.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: integration
+name: Chainlit
+description: Chainlit is an open-source Python package that makes it incredibly fast to build, test and share LLM apps. Integrate the Chainlit API in your existing code to spawn a ChatGPT-like interface in minutes. Visualise and debug your agent's intermediary steps!
+authors:
+    - name: Chainlit Team
+      socials:
+        github: Chainlit
+        twitter: chainlit_io
+pypi: https://pypi.org/project/chainlit/
+repo: https://github.com/Chainlit/chainlit
+type: Custom Node
+report_issue: https://github.com/Chainlit/chainlit/issues
+---
+
+# Chainlit
+
+With a simple line of code, you can leverage Chainlit to interact with your agent, visualise intermediary steps, debug them in an advanced prompt playground and share your app to collect human feedback. More info on the [documentation](https://docs.chainlit.io/).
+
+![Screenshot](./chainlit-haystack.png)
```

Comment:
Let's put the image in a folder called /images so that the /integrations directory is only the md files